mnemonic-core/src/test/java/org/apache/mnemonic/PMDKVolatileMemAllocatorNGTest.java

```diff
@@ -60,7 +60,7 @@ public void testMemChunk() {
     for (int idx = 1; idx <= 5; ++idx) {
       int size = randomGenerator.nextInt(1024 * 1024) + 1024 * 1024;
       mch = act.createChunk(size);
-      System.out.println(String.format("[Seq.%d] addr : %X", idx, size, mch.get()));
+      Assert.assertNotNull(mch, String.format("[Seq.%d] size: %d ", idx, size));
```

Comment:
Please try to add "import org.testng.Assert" in the head of this source file, thanks.